R/quo.R

```diff
@@ -97,8 +84,8 @@ quo_not_missing <- function(x) {
 #' @param quosures *Deprecated*, please use `expressions` instead.
 #'
 #'
-#' @keywords quo
-#' @family quo
+#' @keywords deprecated
+#' @family deprecated
```

Comment:
Should not be changed as the function is not deprecated but only one of its arguments.